src/SeaIceDynamics/SeaIceDynamics.jl

```diff
@@ -0,0 +1,35 @@
+module SeaIceDynamics
+
+export ExplicitMomentumSolver, ExplicitViscoPlasticRheology
+
+using ClimaSeaIce
+
+using Oceananigans
+using KernelAbstractions: @kernel, @index
+using Oceananigans.Utils: launch!
+using Oceananigans.Operators
+using Oceananigans.Grids
+using Oceananigans.Grids: architecture
+
+using Adapt
+
+# The only function we need to provide from the SeaIceDynamics.jl module
+# is the `step_momentum!` function. This module assumes that any model that uses
+# SeaIceDynamics has 
+# - 2D velocities : `u` and `v`
+# - 2D ocean velocities : `u` and `v` (at the surface)
+# - 2D thickness field : `h`
+# - 2D concentration field `ℵ`
+# - a sea-ice density (Float)
+
+# Ice volume
+@inline ice_mass(i, j, k, grid, h, ℵ, ρ) = @inbounds h[i, j, k] * ℵ[i, j, k] * ρ
+
+include("nothing_dynamics.jl") # nothing dynamics, sea-ice velocity is zero!
```

Comment:
Why not just do a PR for nothing dynamics first?

Reply:
right, #24 covers that case